### PR TITLE
database week 1

### PR DESCRIPTION
**Pull request by alaashhadat**

### CODE REVIEW

File: Week1/index.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+  );`;
+db.query(createCitiesTable, function(error) {
+  if (error) throw error.message;
+  console.log('cities table is created successfully!');
+});
+
+// insert data in to countries table
+const insertIntoCountriesTable =
+  'INSERT INTO countries (Code,Name,Continent,Surface_Area,Indep_Year,Population,Life_Expectancy,GNP,GNP_Old,Local_Name,Government_Form,Head_Of_State,Capital,Code2,PRIMARYـKEY) VALUES ?';
```
Comment (harditsingh) [inline Week1/index.js]: You wrote 'Indep_Year' here instead if 'Index_Year'

File: Week1/index.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+db.query('SELECT Name,   Surface_Area FROM countries ORDER BY  Surface_Area DESC ', function(
+  error,
+  resultQuery5,
+) {
+  if (error) throw error;
+  console.log(JSON.parse(JSON.stringify(resultQuery5)));
+});
+
+//names of all the cities in the Netherlands
+db.query('SELECT name  FROM cities WHERE Country_Code="Netherlands"'),
```
Comment (harditsingh) [inline Week1/index.js]: This is the correct query:
`SELECT name  FROM cities WHERE District="Netherlands"`

File: Week1/index.js [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+  password: 'hyfpassword',
+  database: 'userdb',
+});
+db.connect(err => {
+  if (err) {
+    throw err;
+  }
+  console.log('MySql Connected...');
+});
+db.query('CREATE DATABASE IF NOT EXISTS world;', error => {
```
Comment (harditsingh) [inline Week1/index.js]: You can use 'new_world' instead of 'world' here to use the data being loaded by world.sql

File: Week1/index.js [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+  if (error) throw error;
+  console.log('database created successfully');
+});
+db.query('USE WORLD', function(error) {
```
Comment (harditsingh) [inline Week1/index.js]: You can use 'new_world' instead of 'world' here to use the data being loaded by world.sql

**Comment by wouterkleijn** [comment]
Thanks @harditsingh Have a look at this when you have some time @alaashhadat - thanks!

File: Week1/index.js (hunk 4 above)
Comment (alaashhadat) [reply Week1/index.js]: Thanks for your feedback.

**Comment by alaashhadat** [comment]
@wouterkleijn I have addressed the requested changes.

**Comment by unmeshvrije** [comment]
reviewed already.

**Pull request closed without merging**